### PR TITLE
Convert Mac OS Roman punctuation to UTF-8

### PR DESCRIPTION
**Pull request by zeamador**
This PR is the result of running `iconv -f mac -t utf-8` on the A1 source files. It's a large diff, but it should just be changes in comment punctuation -- mostly correcting the apostrophe `’` as well as the occasional lowercase delta `∂`, infinity sign `∞`, and some other characters in various places.

### CODE REVIEW

File: Source_Files/Network/RingGameProtocol.cpp [hunk 1]
```diff
@@ -60,7 +60,7 @@ struct NetStatus
 
 	bool receivedAcknowledgement; /* true if we received a valid acknowledgement for the last ring packet we sent */
 	bool canForwardRing;
-	bool clearToForwardRing; /* true if we are the server and we got a valid ring packet but we didn’t send it */
+	bool clearToForwardRing; /* true if we are the server and we got a valid ring packet but we didn‚Äôt send it */
```
Comment (zeamador) [inline Source_Files/Network/RingGameProtocol.cpp:63]: Note: looks like I messed up with RingGameProtocol.cpp, I need to revert this file.

File: Source_Files/RenderMain/RenderRasterize_Shader.cpp [hunk 2]
```diff
@@ -1148,7 +1148,7 @@ void RenderRasterize_Shader::render_viewer_sprite_layer(RenderStep renderStep)
         rect.Opacity = 1;
 
         /* get_weapon_display_information() returns true if there is a weapon to be drawn.  it
-           should initially be passed a count of zero.  it returns the weaponÕs texture and
+           should initially be passed a count of zero.  it returns the weapon√ïs texture and
```
Comment (zeamador) [inline Source_Files/RenderMain/RenderRasterize_Shader.cpp:1151]: Note: need to fix this file.

**Comment by tst2005** [comment]
Hello guys,
I see 2 kind of fix.
- cosmetic's fixes: about the apostrophe in roman or UTF-8 ... why not just fix it with a simple-stupid ascii `'` ?
- maybe not-cosmetic: I see some [filetype value](https://github.com/Aleph-One-Marathon/alephone/pull/240/commits/e87e20ce0216291ae1390ddb0104f83541f1e27b#diff-bc4cd991b8e8dd14aaf0e22c976d73d288124f60929fafccf180b35418d55e88) ... I don't know that is the origin Mac value. It seems more than just cosmetic, no ?

**Comment by zeamador** [comment]
Hi,
> * cosmetic's fixes: about the apostrophe in roman or UTF-8 ... why not just fix it with a simple-stupid ascii `'` ?
this is the result of automatic conversion. Jason (inconsistently) used the curly apostrophe instead of `'`, so this is reflected now instead of seeing garbage characters.

> * maybe not-cosmetic: I see some [filetype value](https://github.com/Aleph-One-Marathon/alephone/pull/240/commits/e87e20ce0216291ae1390ddb0104f83541f1e27b#diff-bc4cd991b8e8dd14aaf0e22c976d73d288124f60929fafccf180b35418d55e88) ... I don't know that is the origin Mac value. It seems more than just cosmetic, no ?
This is still just a change in a comment, not code. I believe the code has long since been changed so the filetype suffix is `A` instead of `∞`.

**Comment by treellama** [comment]
I will merge this after 1.4 is tagged, if you want to do one last update

**Pull request merged**